zones.txt

```diff
@@ -3331,8 +3331,9 @@ ac.ls
 co.ls
 gov.ls
 net.ls
+nul.ls
 org.ls
-parliament.ls
```

Comment:
What happened here? Did it ever exist? Should we just mark it as obsolete?

Reply:
Yeah obsolete, didn’t realize that was an option. It’s not on the Registry site or Wikipedia, and the only result under it on Google is www.

Reply:
Looks like we don't have an `obsolete` tag (though maybe we should!).